projects/aws/rolesanywhere-credential-helper/Makefile

```diff
@@ -0,0 +1,26 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.18"
+
+REPO=rolesanywhere-credential-helper
+REPO_OWNER=aws
+
+BINARY_TARGET_FILES=aws_signing_helper
+SOURCE_PATTERNS=./cmd/aws_signing_helper
+EXTRA_GO_LDFLAGS=-X k8s.io/component-base/version.gitVersion=$(GIT_TAG)
+
+HAS_S3_ARTIFACTS=true
```

Comment:
How is the binary getting used?

Reply:
It will be added to the credential packages images as a dependency